examples/205-automl.ipynb

```diff
@@ -3,7 +3,11 @@
   {
```

Comment:
<p>Now we can use these parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Done